models/workday.yml

```diff
@@ -91,17 +91,17 @@ models:
       - name: email_address
         description: '{{ doc("email_address") }}' 
 
-      - name: employee_type
-        description: '{{ doc("employee_type") }}' 
+      - name: employee_type 
```

Comment:
There are a few fields that were renamed or missed in the documentation. Please review the docs and the highlighted fields within the workday.yml to see which fields need to be updated.

Reply:
Should be all accounted for now.